core/src/main/scala/besom/internal/codecs.scala

```diff
@@ -70,6 +70,29 @@ trait Decoder[A]:
       }
     override def mapping(value: Value, label: Label): B = ???
 
+object NameUnmangler:
+  private val mangledAnyRefMethodNames = Set(
```

Comment:
We should also mangle and unmangle property names `urn` and `id`

Reply:
There's a separate task for that actually https://github.com/VirtusLab/besom/issues/198